### PR TITLE
[17.0][MIG] sale_partner_incoterm: Migration to 17.0

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: sale_partner_incoterm/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    incoterm_address_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Incoterm Address",
+    )
+
+    @api.onchange("partner_id")
+    def onchange_partner_id_set_incoterm(self):
```
Comment (rousseldenis) [inline sale_partner_incoterm/models/sale_order.py:16]: @StefanRijnhart As Odoo changed sale orders to full computed fields, shouldn't we use them too (store=True, readonly=False) ?

Comment (StefanRijnhart) [reply sale_partner_incoterm/models/sale_order.py:16]: I considered this at the time of migration, but the `incoterm` field is not owned by this module and adding a compute method for it does not scale to more than one module so I am not sure that is the best practice in this case. BTW there are plenty of leftover onchange methods in core Odoo.

**Review by Deriman-Alonso — APPROVED** [review]
LGTM 👍

**Review by traitrantrobz — APPROVED** [review]
The test file and functional is LGTM 👍

**Review by traitrantrobz — CHANGES_REQUESTED** [review]
Please fix the commit title. It should follow this rule:
```
[$VERSION}][$CONTENT: MIG/ADD/FIX/BKP] $MODULE: $TITLE
e.g. [17.0][MIG] web: Migration to 17.0
not [MIG][17.0]
not MIG [17.0]
```
Btw, remember rebase commit.

**Comment by StefanRijnhart** [comment]
@traitrantrobz I followed https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch which has the following line:

`git commit -m "[MIG] $module: Migration to 17.0"`

This results in my commit message 

image

I believe this to be correct. Can you agree with it?

**Comment by traitrantrobz** [comment]
> @traitrantrobz I followed https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch which has the following line:
> 
> `git commit -m "[MIG] $module: Migration to 17.0"`
> 
> This results in my commit message
> 
> image
> 
> I believe this to be correct. Can you agree with it?

I mean you should change the title PR. `[17.0][MIG] sale_partner_incoterm: Migration to 17.0`

**Comment by StefanRijnhart** [comment]
> I mean you should change the title PR. `[17.0][MIG] sale_partner_incoterm: Migration to 17.0`

@traitrantrobz Thanks, underscores restored!

**Review by david-s73 — APPROVED** [review]

File: sale_partner_incoterm/models/sale_order.py (hunk 1 above)
Comment (StefanRijnhart) [reply sale_partner_incoterm/models/sale_order.py:16]: @rousseldenis Can you live with the onchange method?

**Review by rousseldenis — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
@traitrantrobz Your 'Request Changes' review is blocking the approved status of this PR. Can you update your review please?

**Review by traitrantrobz — APPROVED** [review]
LGTM 👍

**Pull request merged**